### PR TITLE
fix: (platform) fix for textarea styles breaking

### PR DESCRIPTION
**Pull request by kavya-b**
#### Please provide a link to the associated issue.
#3001 
#### Please provide a brief summary of this pull request.
There were some old style classes that were causing styles to break randomly on switching from core pages to platform textarea. Removing the unnecessary style classes.

#### Please check whether the PR fulfills the following requirements

- [x] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/master/CONTRIBUTING.md
- [x] tests for the changes that have been done
- [x] all items on the PR Review Checklist are addressed :
https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist

Documentation checklist:
- [x] Documentation Examples
- [x] Stackblitz works for all examples

### CODE REVIEW

File: libs/platform/src/lib/components/form/text-area/text-area.component.html [hunk 1]
```diff
@@ -2,7 +2,6 @@
     #textareaElement
     fd-form-control
     [compact]="contentDensity == 'compact'"
```
Comment (fkolar) [inline libs/platform/src/lib/components/form/text-area/text-area.component.html:4]: Let's follow the same performance approach we did for other components, have isCompact internal property and set it / cache whatever you need to do so we dont have comparission in here. You can check out stepinput, panel I think

Comment (kavya-b) [reply libs/platform/src/lib/components/form/text-area/text-area.component.html]: @fkolar done, please review again

**Comment by kavya-b** [comment]
@fkolar @droshev apart from the actual issue fix, this PR also
- modifies the  way contentDensity is used just like in panel/step-input component by using config file
- modifies the package.json file to call the platform build scripts from the `build-deploy-library` and `build-pack-library` scripts. This was missed out as I was verifying differently and was not building the schematics at all.

**Comment by InnaAtanasova** [comment]
@fkolar could you review again please?
@kavya-b the build is failing. Could you take a look at it? We agreed not to approve PRs with failing builds.

**Comment by kavya-b** [comment]
> @fkolar could you review again please?
> @kavya-b the build is failing. Could you take a look at it? We agreed not to approve PRs with failing builds.

@InnaAtanasova it was a failing test which seems to have been fixed with the latest rebase. The build is succeeding now.

**Comment by InnaAtanasova** [comment]
> > @fkolar could you review again please?
> > @kavya-b the build is failing. Could you take a look at it? We agreed not to approve PRs with failing builds.
> 
> @InnaAtanasova it was a failing test which seems to have been fixed with the latest rebase. The build is succeeding now.

Sweet. If @fkolar or @KevinOkamoto can approve we will merge it

**Pull request merged**